js/views/callinfoview.js

```diff
@@ -66,13 +66,33 @@
 		'		{{/if}}' +
 		'	</div>' +
 		'{{/if}}' +
-		'{{#if showShareLink}}' +
```

Comment:
Why was this removed?

Reply:
This was not intentional, I have restored the removed section.